src/dnaio/readers.py

```diff
@@ -150,3 +155,10 @@ def __init__(
 
     def __iter__(self) -> Iterator[Sequence]:
         return self._iter
+
+    @property
+    def n_records(self):
+        try:
```

Comment:
```suggestion
    def n_records(self) -> int:
        """The number of records parsed so far"""
        try:
```

Reply:
Done. I renamed it to `number_of_records` as well. Given that I complained about pysam this morning, I should not allow myself to make the same mistakes all over again :man_facepalming: . (Your post on the pysam mailing list had a very good API guide from the QT team: abbreviations are never worth it.)